File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdPackagesInfo.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class RetailIdPackagesInfo {
    @JsonProperty("Packages")
    public List<RetailIdRetailIdPackagesInfoPackagesItem> packages;
    public static class RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem {
    @JsonProperty("CreatedAt")
    public String createdAt;
    @JsonProperty("LabelQuantity")
    public Double labelQuantity;
    @JsonProperty("LabelSet")
    public Integer labelSet;
}public static class RetailIdRetailIdPackagesInfoPackagesItem {
    @JsonProperty("EstimatedBalance")
    public Integer estimatedBalance;
    @JsonProperty("HasQrs")
    public Boolean hasQrs;
    @JsonProperty("IssuanceId")
    public String issuanceId;
    @JsonProperty("Issuances")
    public List<RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem> issuances;
    @JsonProperty("QrCount")
    public Integer qrCount;
    @JsonProperty("RequiresVerification")
    public Boolean requiresVerification;
    @JsonProperty("SiblingCount")
    public Integer siblingCount;
    @JsonProperty("Tag")
    public String tag;
}
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/RetailIdPackagesInfo.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class RetailIdPackagesInfo
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("Packages")]
        public required List<RetailIdRetailIdPackagesInfoPackagesItem> Packages { get; set; }
        public class RetailIdRetailIdPackagesInfoPackagesItem
        {
            [global::System.Text.Json.Serialization.JsonPropertyName("EstimatedBalance")]
            public required int EstimatedBalance { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("HasQrs")]
            public required bool HasQrs { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("IssuanceId")]
            public required string IssuanceId { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("Issuances")]
            public required List<RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem> Issuances { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("QrCount")]
            public required int QrCount { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("RequiresVerification")]
            public required bool RequiresVerification { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("SiblingCount")]
            public required int SiblingCount { get; set; }
            [global::System.Text.Json.Serialization.JsonPropertyName("Tag")]
            public required string Tag { get; set; }
            public class RetailIdRetailIdPackagesInfoPackagesItemIssuancesItem
            {
                [global::System.Text.Json.Serialization.JsonPropertyName("CreatedAt")]
                public required string CreatedAt { get; set; }
                [global::System.Text.Json.Serialization.JsonPropertyName("LabelQuantity")]
                public required double LabelQuantity { get; set; }
                [global::System.Text.Json.Serialization.JsonPropertyName("LabelSet")]
                public required int LabelSet { get; set; }
            }
        }
    }
}
